### DISCUSSION TITLE
MetaScraper options

### DISCUSSION THREAD

**Discussion by Sixpax** [adultmm/AdultMediaManager#41] [General]
I'm just wondering if there is a way to add a site to the initial scraper list, eg, additional to ThePornDB and DATA18.com?  If I'm scanning in a few movies from the one site, I've found that the site specific often gives better information than the generic ones. Currently I have to do them one by one so I can then add the site specific scraper and then search and find again.

Thank you

**Comment by adultmm** [comment]
AdultMediaManager plugin tries to match the scrapers by the full path of the file, and prefer specific scrapers if available. This matching doesn't work for you? Could you give me an example? Thanks.
